app/views/vacancies/index.html.slim

```diff
@@ -16,13 +17,15 @@
     .govuk-grid-column-two-thirds-at-desktop
       = render "vacancies/search/fields_and_button", f: f, form: @form
       = render "vacancies/search/open_filters_button", form: @form
-      #search-results
-        - if @show_map
+      - if @show_map
+        #map-results
```

Comment:
this should be passed to the html_attributes of the component if it's required by the component